edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/EdgeOperator.cs

```diff
@@ -861,6 +894,47 @@ VolumeOptions GetVolumesFromModule(IModule<AgentDocker.CombinedDockerConfig> mod
                 : Option.None<(List<V1Volume>, List<V1VolumeMount>, List<V1VolumeMount>)>();
         }
 
+        private async Task GetOrCreatePersistentVolumeClaim(string name, bool readOnly)
```

Comment:
The function name `GetOrCreate...` seems to suggest that this will return some resource. But this is a _void_ function. Consider renaming this to something like `EnsurePersistentVolume` or something.